modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -2968,7 +2968,11 @@ definition {
 
 			ObjectAdmin.goToLayoutsTab();
 
-			WaitForElementPresent(locator1 = "ObjectAdmin#KEBAB_MENU");
+			while ((IsElementNotPresent(locator1 = "ObjectAdmin#KEBAB_MENU")) && (maxIterations = "5")) {
+				Refresh();
+
+				ObjectAdmin.goToLayoutsTab();
+			}
```

Comment:
From the [error](https://testray.liferay.com/reports/production/logs/2023-05/test-1-31/test-portal-testsuite-upstream(master)/2225/functional-tomcat90-mysql57-jdk8/3/6/LocalFile.CreateObject_CanRelateManyOtherEntriesBothObjects/index.html.gz?authuser=0) I saw in the CI, it's as if the click had worked superficially or as if there wasn't enough time for the kebab to appear even with the WaitForElementPresent function. So this loop is an attempt to fix both possible problems.

Reply:
We generally want to avoid using waits like this because it drastically increases the test runtime. A better approach would be to understand what is causing the kebab to not appear. From the screenshot, the test is clicking on the Layout tab but it isn't actually navigating to it. Can we try adding the layout before changing the title field and publishing the object? 

So it'll roughly look like this:

1. ObjectAdmin.openObjectAdmin();
2. Add Layout and customize it
3. Navigate to Details tab
4. Change the title field
5. Publish

Reply:
Another thing to consider is the test design here. [Line 2967](https://github.com/liferay-bpm-qa/liferay-portal/pull/783/files#diff-0dd33d67ad833580d826bea8e12f417faca5bd70f86f50d9e8f2c27f3d6f13dfR2967) is publishing the object definition but we're actually publishing it again [here](https://github.com/liferay-bpm-qa/liferay-portal/pull/783/files#diff-0dd33d67ad833580d826bea8e12f417faca5bd70f86f50d9e8f2c27f3d6f13dfR3000-R3002). That initial publish is what may be causing this issue, since it's causing the page to become temporarily unresponsive.